Atomic/atomic.py

```diff
@@ -390,6 +390,14 @@ def set_args(self, args):
         if not self.storage:
             self.storage = self.get_atomic_config_item(["default_storage"])
 
+        try:
+            self.verify_image = args.verify
+        except:
+            self.verify_image = None
+
```

Comment:
Should this be false and set by default? Or should we read config file to set the default.

It should definately be a boolean.